blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/PipelineStepImpl.java

```diff
@@ -89,6 +91,15 @@ public String getType() {
         return node.getType().name();
     }
 
+    @Override
+    public String getAnalyticsType() {
+        if (node.getNode() instanceof StepAtomNode) {
```

Comment:
I’d include `StepStartNode` in a perfect world, but we aren’t tracking those yet here in the first place.

Reply:
@abayer I've updated this method to take into account `StepStartNode` as you have suggested.